src/cs/ui/Camera.cs

```diff
@@ -0,0 +1,38 @@
+using Godot;
+using System;
+
+public partial class Camera : Camera2D
+{
+	private Vector2 ZOOM_MIN = new Vector2(0.5f,0.5f);
+	private Vector2 ZOOM_MAX = new Vector2(1f,1f);
+	private Vector2 ZOOM_SPEED = new Vector2(0.2f,0.2f);
+	private Vector2 ZoomVal = new Vector2(1.0f,1.0f);
+	
+	// Called when the node enters the scene tree for the first time.
+	public override void _Ready()
```

Comment:
Overrides that aren't actually implemented can be omitted (we have enough code as it is ^^")